src/t2iapi/device/device_responses.proto

```diff
@@ -73,3 +73,20 @@ message InsertContainmentTreeEntryForSequenceIdResponse{
     string handle = 2;          // descriptor handle which is representing the requested BICEPS CONTAINMENT TREE ENTRY
                                 // in the device's current MDIB
 }
+
+/*
+Response containing a boolean answer.
+ */
+message TrueFalseResponse {
```

Comment:
i am not sure if we should use this manipulation at all, i would prefer using the manipulation
> DisconnectRemovableSubsystem

directly and evaluating the RESULT_NOT_SUPPORTED response

Reply:
DoesDescriptorRepresentARemovableSubsystem() is a manipulation that queries information, while DisconnectRemovableSubsystem() changes the state of the device quite significantly. Independent of the TestCase I do see a usecase for both of them.

We also have GetRemovableDescriptorsOfClass() and RemoveDescriptor(). When following your argumentation, then the first would also be redundant.